controllers/admin/config.go

```diff
@@ -66,6 +67,7 @@ func SetStreamTitle(w http.ResponseWriter, r *http.Request) {
 	if !success {
 		return
 	}
+	go webhooks.SendStreamStatusEvent(models.StreamTitleUpdated)
```

Comment:
@gabek Is triggering the webhook here correct? Or should it be done in `core/data/config.go`. I tried doing it here but got a circular import error because `webhook.go` also uses the `data` package

Reply:
I think here makes sense! `data` should probably not have any side effects internally to that package, so I wouldn't fire anything from there.